microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -100,6 +98,9 @@ When you run the optimized version of the page, the blue icons move faster.  Why
    ![The results of the profile.](../media/evaluate-performance-performance-capture-results.msft.png)
 
```

Comment:
Change from:
The problem with the code is that, in each animation frame, it changes the style for each icon, and then queries the position of each icon on the page. Because the styles changed, the browser doesn't know if each icon position changed, so it has to re-layout the icon in order to compute the new position.
to:
The problem with the non-optimized code is that, in each animation frame, it changes the style for each icon, and then queries the position of each icon on the page. Because the styles changed, the browser doesn't know if each icon position changed, so it has to re-layout the icon in order to compute the new position.

Reply:
change made